### PR TITLE
Add go-xgettext parser

### PR DESCRIPTION
**Pull request by micbar**
# Description

Add go-xgettext parser binary from  owncloud-ci/gettext

### CODE REVIEW

File: latest/Dockerfile.amd64 [hunk 1]
```diff
@@ -12,9 +12,13 @@ ARG BUILD_VERSION
 # renovate: datasource=github-releases depName=transifex/cli
 ENV TRANSIFEX_VERSION="${BUILD_VERSION:-v1.6.5}"
 
+# renovate: datasource=github-releases depName=owncloud-ci/gettext
+ENV GO_GETTEXT_VERSION="v1.0.0-beta.3"
```
Comment (xoxys) [inline latest/Dockerfile.amd64:16]: This syntax is not working. Renovate is using a RegexManager for this, and you have to match the defined regex.

Comment (xoxys) [reply latest/Dockerfile.amd64:16]: By default, pre-releases will be ignored by renovate.

File: latest/Dockerfile.amd64 [hunk 2]
```diff
@@ -12,9 +12,13 @@ ARG BUILD_VERSION
...
+
 RUN apt-get update && \
     apt-get install -y git-core perl gettext liblocale-po-perl liblocale-gettext-perl && \
     curl -SsfL "https://github.com/transifex/cli/releases/download/v${TRANSIFEX_VERSION##v}/tx-linux-amd64.tar.gz" | tar xz -C /usr/local/bin tx && \
+    curl -SsfL "https://github.com/owncloud-ci/gettext/releases/download/v${GO_GETTEXT_VERSION##v}/go-xgettext-${GO_GETTEXT_VERSION##v}-linux-amd64" -o /usr/local/bin/go-xgettext && chmod +x /usr/local/bin/go-xgettext && \
```
Comment (xoxys) [inline latest/Dockerfile.amd64:21]:
```suggestion
    curl -SsfL "https://github.com/owncloud-ci/gettext/releases/download/v${GO_GETTEXT_VERSION##v}/go-xgettext-${GO_GETTEXT_VERSION##v}-linux-amd64" -o /usr/local/bin/go-xgettext && \
    chmod 755 /usr/local/bin/go-xgettext && \
```

File: latest/Dockerfile.amd64 (hunk 1 above)
Comment (micbar) [reply latest/Dockerfile.amd64:16]: my plan was, to release 1.0.0 after merging this to test renovate. How would the regex look like?

Comment (xoxys) [reply latest/Dockerfile.amd64:16]: https://github.com/owncloud-ci/renovate-presets/blob/master/docker.json#L35

Just use the syntax from transifex above, e.g.

```
ARG GO_GETTEXT_VERSION

# renovate: datasource=github-releases depName=owncloud-ci/gettext
ENV GO_GETTEXT_VERSION="${GO_GETTEXT_VERSION:-v1.0.0-beta.3}"
```

Comment (micbar) [reply latest/Dockerfile.amd64:16]: ah, i see https://github.com/owncloud-ci/renovate-presets/blob/master/docker.json#L35

File: latest/Dockerfile.amd64 [hunk 3]
```diff
@@ -8,13 +8,19 @@ LABEL org.opencontainers.image.source="https://github.com/owncloud-ci/transifex"
...
 ENV TRANSIFEX_VERSION="${BUILD_VERSION:-v1.6.5}"
 
+# renovate: datasource=github-releases depName=owncloud-ci/gettext
+ENV GO_GETTEXT_VERSION="${GO_GETTEXT_VERSION:-v1.0.0-beta.3}"
+
 RUN apt-get update && \
     apt-get install -y git-core perl gettext liblocale-po-perl liblocale-gettext-perl && \
     curl -SsfL "https://github.com/transifex/cli/releases/download/v${TRANSIFEX_VERSION##v}/tx-linux-amd64.tar.gz" | tar xz -C /usr/local/bin tx && \
+    curl -SsfL "https://github.com/owncloud-ci/gettext/releases/download/v${GO_GETTEXT_VERSION##v}/go-xgettext-${GO_GETTEXT_VERSION##v}-linux-amd64" -o /usr/local/bin/go-xgettext && \
+    chmod +x /usr/local/bin/go-xgettext && \
```
Comment (xoxys) [inline latest/Dockerfile.amd64:23]:
```suggestion
    chmod 755 /usr/local/bin/go-xgettext && \
```

**Review by xoxys — APPROVED** [review]

**Pull request merged**